packages/core/src/App/Containers/Layout/header/dashboard-header.jsx

```diff
@@ -55,60 +50,94 @@ const LoggedInHeader = ({ is_dark_mode }) => {
 const LoggedOutHeader = () => {
     const history = useHistory();
     const { is_dashboard } = React.useContext(PlatformContext);
+    const [active_dropdown, setCurrentDropdown] = React.useState('');
+    const [active_link_ref, setActiveLinkRef] = React.useState(null);
+    const [active_dropdown_ref, setActiveDropdownRef] = React.useState(null);
+    const nav_dropdown_ref = React.useRef(null);
+
+    const handleDropdownClick = (dropdown, target) => {
+        setCurrentDropdown(dropdown);
+        if (target) {
+            setActiveLinkRef(target);
+        }
+    };
+
+    const handleOutsideClick = React.useCallback(e => {
```

Comment:
Similarly to `useEffect`, the `useCallback` hook takes a dependency array which should be populated, else it'll be re-created on each render (like before). In this case eslint is probably cool with specifying an empty dependency array (`[]`) so that it is only created after the first render.